File: DevApp.ViewModels/Premium.cs
using DotNetify.Elements;

namespace DotNetify.DevApp
{
   public class DotNetifyTesting : BaseVM
   {
      public string Content => new Markdown("DotNetify.DevApp.Docs.Premium.DotNetifyTesting.md");
   }

   public class DotNetifyLoadTester : BaseVM
   {
      public string Content => new Markdown("DotNetify.DevApp.Docs.Premium.DotNetifyLoadTester.md");
   }

   public class DotNetifyObserver : BaseVM
   {
      public string Content => new Markdown("DotNetify.DevApp.Docs.Premium.DotNetifyObserver.md");
   }

   public class DotNetifyResiliencyAddon : BaseVM
   {
      public string Content => new Markdown("DotNetify.DevApp.Docs.Premium.DotNetifyResiliencyAddon.md");
   }
}

File: DevApp/client/app/views/Premium.js
import React from "react";
import { Markdown, withTheme } from "dotnetify-elements";
import Article from "../components/Article";

export const DotNetifyTesting = withTheme(() => (
  <Article vm="DotNetifyTesting" id="Content">
    <Markdown id="Content" />
  </Article>
));

export const DotNetifyLoadTester = withTheme(() => (
  <Article vm="DotNetifyLoadTester" id="Content">
    <Markdown id="Content" />
  </Article>
));

export const DotNetifyObserver = withTheme(() => (
  <Article vm="DotNetifyObserver" id="Content">
    <Markdown id="Content" />
  </Article>
));

export const DotNetifyResiliencyAddon = withTheme(() => (
  <Article vm="DotNetifyResiliencyAddon" id="Content">
    <Markdown id="Content" />
  </Article>
));
